spalp.module

```diff
@@ -52,6 +52,24 @@ function spalp_entity_view(array &$build, EntityInterface $entity, EntityViewDis
     // TODO: Check if the app's module defines a library?
     // Add the extending module's library to the render array.
     $build['#attached']['library'][] = "$app_id/$app_id";
+
+    // Generate relative url with additional slash removed for default language.
```

Comment:
not sure why we need to do this - wouldn't it make more sense for the JSON endpoint URL to follow the language pattern

i.e. if you're on `/cy/branch-finder`, it would make sense for the JSON endpoint to be on `/cy/branch-finder/json`, or something like that

Reply:
this implementation has the same approach.. i.e. on /cy/node/:nid you will get <link type="application/json" id="appConfig" rel="alternate" href="/cy/spalp-example/spalp-example.json" /> generated on page